6_php/activity_1.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+///exercise1
+
+$s = "This is a string";
+$n = 3.3;
+$b = true;
+$array = ["hitomi", "iroha", "akari"];
+
+echo "<br>".$s;
+echo "<br>".$n;
+echo "<br>".$b;
+echo "<br>".$array;
+
+///exercise2
+
+$hello = "hello world!";
+
+echo "<br>".$hello;
+echo "<br>".$hello[0];
+echo "<br>".$hello[$hello.length-1];
```

Comment:
Please change .length to strlen so instead of

$hello[$hello.length-1]

it would be

$hello[strlen($hello)-1]

strlen is used for counting the length of the string in php